src/test/java/com/microsoft/azure/vmagent/test/ITAzureVMManagementServiceDelegate.java

```diff
@@ -636,7 +654,25 @@ public void removeStorageBlobKeepsContainerIfNotEmpty() {
             Assert.assertTrue(containerExists(blobToBeDeleted));
             Assert.assertFalse(blobExists(blobToBeDeleted));
             Assert.assertTrue(blobExists(notDeletedBlob));
-        } catch (Exception e) {
+        } catch (Exception e){
+            LOGGER.log(Level.SEVERE, null, e);
+            Assert.assertTrue(e.getMessage(), false);
+        }
+    }
+    
+    @Test
+    //Add Test for global first, will add test for mooncake later
```

Comment:
the expect endpoint for public azure is wrong, it's "core.windows.net" instead of "core.widnows.azure.net". Fix it and test pass on my machine for public azure.